.gitignore

```diff
@@ -1,3 +1,5 @@
+mussidae/data/time-range-sources/
```

Comment:
Some of the time range sources that I'm using (Kristina's for instance) can't be checked into a public repo, so I git-ignored this so that I wouldn't accidentally check anything in when I was using this as a staging area. It may be better to just have a different staging area for Kristina's stuff and un-gitignore this.  I'll take a look.